web-animations-1/Overview.bs

```diff
@@ -2296,12 +2344,16 @@ definitions:
 
 An <a>animation effect</a> is in the <dfn>before phase</dfn> if the
 animation effect's <a>local time</a> is not <a>unresolved</a> and
-<em>either</em> of the following conditions are met:
+<em>any</em> of the following conditions are met:
 
 1.  the <a>local time</a> is less than the <a>before-active boundary time</a>,
     <em>or</em>
 1.  the <a>animation direction</a> is &lsquo;backwards&rsquo; and the <a>local
-    time</a> is equal to the <a>before-active boundary time</a>.
+    time</a> is equal to the <a>before-active boundary time</a>,
+    <em>or</em>
+1.  the effect is <a>associated with a timeline</a> <em>and</em> the associated
+    <a>timeline</a>'s [=timeline phase|phase=] is
+    [=timeline before phase|before=]
```

Comment:
Two questions:

* Is this the behavior we want? It effectively means you can't seek an animation associated with a timeline in the before/after phase which seems wrong?
* Do we need to do something similar for the after phase?

Reply:
Deferring adding the details of how phase affects effects until a later PR.